charts/seed-bootstrap/templates/gardener-seed-admission-controller/poddisruptionbudget.yaml

```
@@ -0,0 +1,16 @@
+{{- if gt (int .Values.gardenerSeedAdmissionController.replicas) 1 }}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: gardener-seed-admission-controller
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: gardener
+    role: seed-admission-controller
+spec:
+  maxUnavailable: {{ sub (int .Values.gardenerSeedAdmissionController.replicas) 1 }}
```

Why are you not simply using `minAvailable`?
```suggestion
  minAvailable: 1
```

---

Hm..., too easy to read? 😛Fixed, thanks.